UnitTests/GitUITests/TranslationTest.cs

```diff
@@ -40,9 +40,10 @@ public void CreateInstanceOfClass()
 
                 foreach (var type in types)
                 {
+                    ITranslate obj = null;
                     try
                     {
-                        var obj = (ITranslate)TranslationUtil.CreateInstanceOfClass(type);
+                        obj = (ITranslate)TranslationUtil.CreateInstanceOfClass(type);
```

Comment:
💡  If `ITranslate` implements `IDisposable`, then this can use a `using` statement instead of `try`/`finally`. Otherwise there's not much benefit to making `ITranslate` disposable. 😄

Reply:
I'm pretty sure we can write this as follows and remove the `finally`
```suggestion
                        using ITranslate obj = (ITranslate)TranslationUtil.CreateInstanceOfClass(type);
```

Reply:
Already in progress...
I had the hope I could improve the <censored /> ScriptRunnerTests before.

Reply:
I hope, next time I will remember myself that C# has a (partial) analogon for C++ smart pointers (although `using` does not help with ctor/Dispose unfortunately). 😃